src/applications/personalization/dashboard/utils/constants.js

```diff
@@ -9,3 +9,5 @@ export const FETCH_RECIPIENTS_SUCCESS = 'FETCH_RECIPIENTS_SUCCESS';
 // Folders action types
 export const FETCH_FOLDER_FAILURE = 'FETCH_FOLDER_FAILURE';
 export const FETCH_FOLDER_SUCCESS = 'FETCH_FOLDER_SUCCESS';
+export const FETCH_INBOX_FAILURE = 'FETCH_INBOX_FAILURE';
```

Comment:
I'll update the new action types but not the existing ones as I don't want to expand the scope of this PR too much. Will also update the related action payload to be named `response` instead of `messages` to be in line with out Redux guidelines